src/ui/public/vislib_vis_type/__tests__/_vislib_renderbot.js

```diff
@@ -84,15 +84,15 @@ describe('renderbot', function exportWrapper() {
   });
 
   describe('param update', function () {
-    let params = { one: 'fish', two: 'fish' };
+    let $el = $('<div>testing</div>');
+    let params = { el: $el[0], one: 'fish', two: 'fish' };
```

Comment:
Fairly certain I wrote the original tests. It pleases me so much that someone noticed this reference 🏆 . It's probably not the only place I've used this either...